examples/terraform/openstack/output.tf

```diff
@@ -86,7 +86,7 @@ output "kubeone_workers" {
           securityGroups = [openstack_networking_secgroup_v2.securitygroup.name]
           network        = openstack_networking_network_v2.network.name
           subnet         = openstack_networking_subnet_v2.subnet.name
-          floatingIpPool = var.external_network_name
+          # floatingIpPool = var.external_network_name
```

Comment:
We should probably revert this change?!

Reply:
We decided to keep this change. The release note is updated accordingly.